apps/estuary/lib/estuary_web/live/event_live_view.ex

```diff
@@ -58,6 +58,19 @@ defmodule EstuaryWeb.EventLiveView do
     {:noreply, push_patch(socket, to: Routes.live_path(socket, __MODULE__, search_params))}
   end
 
+  def handle_info(%{search: params}, socket) do
```

Comment:
Did you remove the event subscription part of this in an earlier PR?

Reply:
No